### ISSUE TITLE
HA-Proxy does not restart after `ipconf.sh`

### ISSUE THREAD

**Issue by viccross** [viccross/ansible-playbooks#73]
The HA-Proxy service doesn't restart like it's supposed to as part of `ipconf.sh` processing: 
```
● haproxy.service - HAProxy Load Balancer
   Loaded: loaded (/usr/lib/systemd/system/haproxy.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Mon 2022-01-31 02:09:27 UTC; 3s ago
  Process: 33875 ExecStart=/usr/sbin/haproxy -Ws -f $CONFIG -p $PIDFILE $OPTIONS (code=exited, status=1/FAILURE)
  Process: 33869 ExecStartPre=/bin/bash -c until host api.ocp-z-poc.wsc.ibm; do sleep 1; done && /usr/sbin/haproxy -f $CONFIG -c -q (code=exited, status=0/SUCCESS)
 Main PID: 33875 (code=exited, status=1/FAILURE)

Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [WARNING] 030/020927 (33875) : config : 'option forwardfor' ignored for backend 'ocp4-router-http' as it requires HTTP mode.
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [WARNING] 030/020927 (33875) : config : 'option forwardfor' ignored for backend 'ocp4-router-https' as it requires HTTP mode.
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [WARNING] 030/020927 (33875) : Setting tune.ssl.default-dh-param to 1024 by default, if your workload permits it you should set >
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [ALERT] 030/020927 (33875) : Starting frontend ocp4-kubernetes-api-server: cannot bind socket [9.60.86.72:6443]
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [ALERT] 030/020927 (33875) : Starting frontend ocp4-machine-config-server: cannot bind socket [9.60.86.72:22623]
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [ALERT] 030/020927 (33875) : Starting frontend ocp4-router-http: cannot bind socket [9.60.86.72:80]
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm haproxy[33875]: [ALERT] 030/020927 (33875) : Starting frontend ocp4-router-https: cannot bind socket [9.60.86.72:443]
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm systemd[1]: haproxy.service: Main process exited, code=exited, status=1/FAILURE
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm systemd[1]: haproxy.service: Failed with result 'exit-code'.
Jan 31 02:09:27 lxocpb01.ocp-z-poc.wsclab.endicott.ibm systemd[1]: Failed to start HAProxy Load Balancer.
```
The process cannot bind its configured ports because it's using the wrong IP address.

**Comment by viccross** [comment]
The config file `/etc/haproxy/haproxy.cfg` is not edited after the domain name change, and is still pointing to the `wsc.ibm` domain.  Actually, the DNS config doesn't look to have changed at all. ☹️

**Comment by viccross** [comment]
Issue was that the new zone file for Finna support was missing from the ELAN, which meant that the `update-dns-domain` playbook (which does all the work) did not run.

Fixed now in commit 15f0e1b.
